### DISCUSSION TITLE
schemas 파일을 수정시 '퍼블리시 환경'에서 서버가 실행되지 않습니다.

### DISCUSSION THREAD

**Discussion by Cha-SeongHun** [naverz/zepeto-studio-kor#543] [Issue Report (Legacy)]
schemas 파일을 수정하면 퍼블리시 환경에서 서버가 서비스되지 않습니다.
증상으로는 RoomJoined가 호출되지 않고, 채팅에 접속되었다는 안내메세지가 표시되지 않습니다.

PC및 QR코드를 통한 모바일 환경에서는 schemas 파일의 변경여부와 관계없이 정상동작하지만
프로젝트를 테스트 계정에 업로드하고 모바일로 접속할 경우에만 재현됩니다.

정상 동작이 되는 유일한 schemas세팅은 아래와 같습니다.
```
//Room State
map Player "players"

// Schema Types
Room State
  Player
    ref Transform "transform"
  Transform
    ref Vector3 "transform"
    ref Vector3 "rotation"
  Vector3
    number "x"
    number "y"
    number "z"
```

시스템 정보는 Windows11 (AMD시스템), Android 12 (갤럭시 Z폴드2), 제페토 패키지 버전은 1.1.5 입니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 채팅에 접속되었다는 안내 메세지가 뜨지 않은 것은 비정상적인 상황으로 보여집니다.
수정하신 스키마, index 파일을 같이 올려주시면 확인이 용이할 것 같습니다.

**Reply by Cha-SeongHun** [reply]
index파일은 프로젝트 생성후 수정하지 않은 상태였으며
스키마는 아래와 같이 수정할 경우 재현되었습니다.

```
{
"State" : {"players" : {"map" : "Player"}},
"Player" : {"transform" : "Transform"},
"Transform" : {"transform" : "Vector3","rotation" : "Vector3"},
"Vector3" : {"x" : "number","y" : "number","z" : "number"},
"Test" : {"text" : "string"}
}
```

**Reply by Zepeto-TechEvangelist** [reply]
스키마를 파일에서 수정하면 반영이 제대로 되지 않습니다.
인스펙터에서 추가하신 후 Apply 해보시겠어요?

**Reply by Cha-SeongHun** [reply]
> 스키마를 파일에서 수정하면 반영이 제대로 되지 않습니다. 인스펙터에서 추가하신 후 Apply 해보시겠어요?

json파일을 수정하지 않고 에디터 인스펙터로 작엽 후 Apply버튼을 눌렀습니다

**Reply by hyeonjin-jo** [reply]
안녕하세요 @Cha-SeongHun 님 혹시 사용하신 월드아이디를 알려주실 수 있을까요?

**Reply by Cha-SeongHun** [reply]
> 안녕하세요 @Cha-SeongHun 님 혹시 사용하신 월드아이디를 알려주실 수 있을까요?

이메일을 기재해주시면 그쪽으로 보내드리겠습니다.

**Reply by hyeonjin-jo** [reply]
넵 hyeonjin.jo@naverz-corp.com 으로 부탁드립니다!

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 해당 이슈가 월드 출시 이후에 발생하였는지 확인 부탁 드립니다.
